config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -2,7 +2,7 @@ presubmits:
   kubernetes/kubernetes:
   - name: pull-kubernetes-bazel-build
     decorate: true
-    always_run: true
+    always_run: false
```

Comment:
this seems unrelated?

Reply:
this is a change that would require I also update gubernator's config, so this PR _should_ be failing

this is how I've discovered that bazel doesn't invoke ./hack/verify-all.sh